app/diagnostics.go

```diff
@@ -8,9 +8,10 @@ import (
 	"runtime"
 	"strings"
 
+	"github.com/segmentio/analytics-go"
```

Comment:
Why this change?

Reply:
it automatically changed by `goimport`, because `external` import must be in different block with `internal` import or `builtin` import